### DISCUSSION TITLE
Need to know if the Azure log analytics workspace table schema can be used for my fluentd customlogs table

### DISCUSSION THREAD

**Discussion by karthikeyan-br** [fluent/fluentd#4499] [General]
Current Setup:
There are 3 AKS present at the moment and logs from the clusters are being collected by fluentbit daemonset and is being sent to a fluentd container instance. From fluentd the logs are being sent to a log analytics workspace. The logs are currently being stored in a custom logs table.

Requirement:
I need to send different types of logs from these clusters(eg., audit logs, container logs) to different tables in the same log analytics workspace. Is this possible? If so, how? 

Also, will it be possible to send the logs from fluentd to default tables that azure creates when using a diagnostic setting instead of the custom log table.

Could somone help me with this.

**Comment by daipom** [comment]
You can route Fluentd events to different Output settings depending on their tags:

* https://docs.fluentd.org/configuration/config-file#id-2.-match-tell-fluentd-what-to-do

Is this information likely to resolve the issue?
